### DISCUSSION TITLE
Decky PowerToolls problem

### DISCUSSION THREAD

**Discussion by HorridSage12** [SteamDeckHomebrew/decky-loader#215] [Q&A] [answered]
Hello, some days ago I was changing some parameters in PowerTools plugin, and unfortunately I "broke" the plugin. Now every time I try to install and reinstall it, it crashes as soon as I enter the gaming mode of Steam Deck, I can't install any plugin, I tried deleting plugins from the homebrew folder, but to no avail. Is there any way to fix the problem and reset the plugin settings?

**Comment by NGnius** [comment, marked as answer]
Try deleting your `~/.config/powertools` folder (or just the contents of it). You'll probably need superuser permissions to do that.

**Reply by HorridSage12** [reply]
Unfortunately, I don't have any folder named powertools in .config

**Reply by HorridSage12** [reply]
Ok, I solved it by deleting the hidden file in the services folder

thanks for your help

**Reply by HorridSage12** [reply]
> Try deleting your `~/.config/powertools` folder (or just the contents of it). You'll probably need superuser permissions to do that.
